README.md

```diff
@@ -1,15 +1,15 @@
 # Overview
 
-This project contains sample source code that demonstrates how to integrate true[X]'s Roku
-ad renderer. This further exemplifies the needed logic to manage true[X] opt-in flows (choice cards) as 
-fully stitched into the video stream. 
+This project contains sample source code that demonstrates how to integrate true[X]'s Roku ad renderer in
+combination with RAF for client side ads. RAF's responsibilities here are to parse an ad response, detect
+if there are any ads to play, and to play any non-truex ads
 
 For a more detailed integration guide, please refer to: https://github.com/socialvibe/truex-roku-integrations.
 
 # Implementation Details
 
-In this project we simulate the integration with a live Ad Server (CSAI use case) or SSAI provider through a mock ad playlist configuration. This is meant to capture the stream's ad pods, including their duration and the reference to the true[X] payloads in each pod. This configuration is maintained in `res/reference-app-streams.json` as part of the `vmap` key. In this sample channel, two ad breaks are defined, `preroll` and `midroll-1`. The duration of the true[X] specific Choice Card video asset is called out as well as the duration of the standard video ads. This is a simplified representation of what would otherwise come through a provider-dependent XML or JSON syntax, but should be sufficient to exemplify the flow. The stream location itself is maintained in the `url` value.
+For the purpose of this project, some of the data is hard coded as the primary goal here is to showcase the RAF - TrueX Ad integration for client side handled ads. The core of this implementation starts in `PlaybackTask`. As a brief summary, the app will launch and prepare the TrueX Ad Renderer (TAR). Once ready, the app will launch to a simple page (`DetailsFlow`), and waits for the user to start playback by selecting the one/play action. Upon selecting the action, the `RafContentFlow` is started, which sets up some UI elements, and then invokes `PlaybackTask` where most of the action starts.
```

Comment:
Should be more like "As a brief summary, the app will **load** and prepare the TrueX Ad Renderer (TAR) **component library**. Once ready..."

The way it was worded makes it sound like a TAR instance is created on channel launch or super early in the app lifecycle while actually it's right in line with processing the results of ad requests. I think you meant to speak to the component library load actually so revising this makes sense.